test/test_split.cpp

```diff
@@ -19,6 +19,7 @@
 #include "rcutils/error_handling.h"
 #include "rcutils/split.h"
 #include "rcutils/types/string_array.h"
+#include "rcutils/types/char_array.h"
```

Comment:
Fix ordering

Reply:
That include actually shouldn't be there! It's a leftover, removed in https://github.com/ros2/rcutils/pull/272/commits/46a4c15b806cfe6ec1990260faed106e37763339.